modules/github_oidc_roles/main.tf

```diff
@@ -0,0 +1,68 @@
+locals {
+  auth_domain = "token.actions.githubusercontent.com"
+  org_name    = "ministryofjustice"
+  permission_list = [
```

Comment:
I think this might be better as an input variable where you pass in `"repo:${local.org_name}/${var.repository}:pull_request"` and `"repo:${local.org_name}/${var.repository}:ref:refs/heads/*"`. This gives us the chance to limit what github actions triggers the role can be used for.

Reply:
Sounds sensible - i've pushed an update to make the permissions a var with a default that matches the current setup